tests/js/server/aql/aql-functions-date.js

```diff
@@ -2457,6 +2457,37 @@ function ahuacatlDateFunctionsTestSuite () {
 	  assertEqual([ "2020-02-29T23:00:00.000Z" ], 
 		getQueryResults("RETURN DATE_LOCALTOUTC(DATE_ADD(DATE_UTCTOLOCAL('2020-01-31T23:00:00.000Z', 'Europe/Berlin'), 1, 'months'), 'Europe/Berlin')"));	  
     },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezone
+    // //////////////////////////////////////////////////////////////////////////////
+
+    testDateTimeZone: function () {
+	  var systemtz = null;
+	  var res = getQueryResults("RETURN DATE_TIMEZONE()", {})[0];
+	  
+	  try {
+	    systemtz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+	  } catch {
+	  }
+		
+	  if (systemtz) {
+	    assertEqual(systemtz, res);
+	  } else {
+		assertTrue(res != null);
+	  }
+    },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezones
+    // //////////////////////////////////////////////////////////////////////////////
+	
+	testDateTimeZones: function () {
+	  var res = getQueryResults("RETURN DATE_TIMEZONES()", {})[0];
+	  assertTrue(res.indexOf("America/New_York") !== -1);
+	  assertTrue(res.indexOf("Europe/Berlin") !== -1);
+	  assertTrue(res.indexOf("Asia/Shanghai") !== -1);
```

Comment:
Same here, using `assertNotEqual` is a bit more descriptive:
```suggestion
	  assertNotEqual(-1, res.indexOf("America/New_York"));
	  assertNotEqual(-1, res.indexOf("Europe/Berlin"));
	  assertNotEqual(-1, res.indexOf("Asia/Shanghai"));
```

Reply:
Hi, thanks can you please verify that these tests run?
or is there any way to just execute the js aql tests without the cygwin shell under windows?

Reply:
Yes, I can verify the tests on this branch later today.
I guess there is a way to execute these tests under Windows too, but as I am a Linux user that is beyond my own experience. Let me try to find someone to add instructions here...

Reply:
Do you have [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) installed? IMHO that is the easiest way to run the tests on windows. In WSL you can simply do `ARANGOSH=path/to/arangosh.exe ./scripts/unittests shell_server_aql`

If you don't have WSL (and dont want to install it), you should be able to call arangosh directly just like the unittest script does:
```
path\to\arangosh.exe -c etc\arangodb3\arangosh.conf --log.level warning --server.endpoint none --javascript.allow-external-process-control true --javascript.execute UnitTests\unittest.js -- shell_server_aql
```